packages/snaps-controllers/src/test-utils/cronjob.ts

```diff
@@ -26,28 +26,31 @@ export const MOCK_CRONJOB_PERMISSION: PermissionConstraint = {
   parentCapability: SnapEndowments.Cronjob,
 };
 
-type GetCronjobPermissionArgs = {
-  expression?: string;
-};
+type GetCronjobPermissionArgs =
+  | {
+      expression?: string;
+    }
+  | {
+      duration?: string;
+    };
 
 /**
  * Get a cronjob permission object as {@link PermissionConstraint}.
  *
  * @param args - The arguments to use when creating the permission.
- * @param args.expression - The cron expression to use for the permission.
  * @returns The permission object.
  */
-export function getCronjobPermission({
-  expression = '59 6 * * *',
-}: GetCronjobPermissionArgs = {}): PermissionConstraint {
+export function getCronjobPermission(
+  args: GetCronjobPermissionArgs = {},
```

Comment:
```suggestion
  args: GetCronjobPermissionArgs,
```

Should we remove the default empty argument here to not accidentally create invalid permissions?